app/views/passengers/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <div class="div3">
+    <h3 id="pass_list" class="passenger_list">List of Passengers</h3>
```

Comment:
Just a small comment-- it happens to be CSS/HTML naming convention that class names are hyphen separated. I know! So many different naming conventions.